lightning/src/ln/offers_tests.rs

```diff
@@ -727,6 +727,82 @@ fn creates_and_pays_for_offer_using_one_hop_blinded_path() {
 	expect_recent_payment!(bob, RecentPaymentDetails::Fulfilled, payment_id);
 }
 
+/// Verifies that the Invoice Request Message can be retried if it fails to reach the
```

Comment:
nit: Only capitalize words that are the first word in a sentence, proper nouns, or is an identifier with the same capitalization.

Likewise in other comments / log messages

Reply:
Done! [pr3010.17](https://github.com/shaavan/rust-lightning/commits/pr3010.17)
Thanks!